### PR TITLE
Adding Java Workshop #3 video

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tihomir@temporal.io>

## What does this PR do?

* Adds third java workshop youtube link
* Adds small descriptions of what each workshop topics

### CODE REVIEW

File: docs/java/testing-and-debugging.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ public class HelloActivityTest {
 For Junit4 tests, Temporal provides the TestWorkflowRule class which simplifies the Temporal test environment setup, as well as the
 creation and shutdown of Workflow Workers in your tests.
 
-To start using JUnit4 TestWorkflowRule in your tests using [Gradle]((https://gradle.org/)), you need to enable capability [`io.temporal:temporal-testing-junit4`]:
+To start using JUnit4 TestWorkflowRule in your tests using [Gradle](<(https://gradle.org/)>), you need to enable capability [`io.temporal:temporal-testing-junit4`]:
```
Comment (rachfop) [inline docs/java/testing-and-debugging.md:152]:
```suggestion
To start using JUnit4 TestWorkflowRule in your tests using [Gradle](https://gradle.org/), you need to enable capability [`io.temporal:temporal-testing-junit4`]:
```

Removing extra syntax.

File: docs/java/testing-and-debugging.md [hunk 2]
```diff
@@ -195,7 +195,7 @@ public class HelloActivityJUnit4Test {
 For Junit5 tests, Temporal also provides the TestWorkflowExtension helped class which can be used to simplify the Temporal test environment setup
 as well as Workflow Worker startup and shutdowns.
 
-To start using JUnit5 TestWorkflowExtension in your tests with [Gradle]((https://gradle.org/)), you need to enable capability [`io.temporal:temporal-testing-junit5`]:
+To start using JUnit5 TestWorkflowExtension in your tests with [Gradle](<(https://gradle.org/)>), you need to enable capability [`io.temporal:temporal-testing-junit5`]:
```
Comment (rachfop) [inline docs/java/testing-and-debugging.md:198]:
```suggestion
To start using JUnit5 TestWorkflowExtension in your tests with [Gradle](https://gradle.org/), you need to enable capability [`io.temporal:temporal-testing-junit5`]:
```

File: docs/java/index.md [hunk 3]
```diff
@@ -57,9 +57,26 @@ The SDK contains the following tools:
...
+  - Dynamic Workflows / Activities
+- [Java SDK Workshop #3](https://www.youtube.com/watch?v=8DFox0fGjzI)
+  - Workflow and Activity Types, Signal and Query Names
+  - Different types of Stubs
+  - Dynamic Signal/Query Handlers
+  - ContinueAsNew and Signals
+  - Client Error Handling
+  - Setting up client SSL/mTLS
 
 ## The basics
```
Comment (flossypurse) [inline docs/java/index.md:81]: To bring this page up to parity - we can delete this whole section.

File: docs/java/index.md [hunk 4]
```diff
@@ -57,9 +57,26 @@ The SDK contains the following tools:
...
+  - Versioning
+  - Error Handling
+  - Dynamic Workflows / Activities
+- [Java SDK Workshop #3](https://www.youtube.com/watch?v=8DFox0fGjzI)
+  - Workflow and Activity Types, Signal and Query Names
+  - Different types of Stubs
+  - Dynamic Signal/Query Handlers
+  - ContinueAsNew and Signals
+  - Client Error Handling
+  - Setting up client SSL/mTLS
```
Comment (flossypurse) [inline docs/java/index.md:79]: I think we should let YouTube do what YouTube is good at, and just have a general "Java SDK playlist" link -> https://www.youtube.com/playlist?list=PLl9kRkvFJrlQ8KsM6m9cFfCeQegq_B8x4

Comment (flossypurse) [inline docs/java/index.md:79]: I think we should let YouTube do what YouTube is good at, and just have a general "Java SDK playlist" link -> https://www.youtube.com/playlist?list=PLl9kRkvFJrlQ8KsM6m9cFfCeQegq_B8x4

**Comment by flossypurse** [comment]
I put up a link to the Java SDK playlist here and refactored this page to be similar to the Go one https://github.com/temporalio/documentation/pull/1079

**Comment by flossypurse** [comment]
Closing in favor of these changes: https://github.com/temporalio/documentation/pull/1079

**Pull request closed without merging**